docs/description/description.json

```diff
@@ -36,227 +36,37 @@
 }, {
   "patternId" : "B108",
   "title" : "hardcoded_tmp_directory",
-  "description" : "B108: Test for insecure usage of tmp file/directory",
+  "description" : "Buffer()",
   "parameters" : [ ]
 }, {
   "patternId" : "B109",
   "title" : "Test for a password based config option not marked secret",
-  "description" : "B109: Test for a password based config option not marked secret",
+  "description" : "Buffer()",
   "parameters" : [ ]
 }, {
   "patternId" : "B110",
   "title" : "try_except_pass",
-  "description" : "B110: Test for a pass in the except block",
+  "description" : "Buffer()",
```

Comment:
That "new NodeBuffer" in the doc generator has a different effect than what you expect.